examples/kubernetes-ingress/scripts/03-install-kubernetes-worker.sh

```diff
@@ -11,12 +11,28 @@ dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 source "${dir}/helpers.bash"
 
 cache_dir="${dir}/../../../hack/cache"
-
 k8s_cache_dir="${cache_dir}/k8s/${k8s_version}"
+certs_dir="${dir}/certs"
 
-log "Installing kubernetes worker components..."
+function install_crio() {
+  sudo apt-key adv --recv-key --keyserver keyserver.ubuntu.com 8BECF1637AD8C79D
 
-certs_dir="${dir}/certs"
+   cat <<EOF > /etc/apt/sources.list.d/projectatomic-ubuntu-ppa-artful.list
+deb http://ppa.launchpad.net/projectatomic/ppa/ubuntu xenial main
+deb-src http://ppa.launchpad.net/projectatomic/ppa/ubuntu artful main
+EOF
+   sudo apt-get update
+   sudo apt-get install cri-o-1.10 -y
+
+   download_to "${cache_dir}/containerd" "cri-containerd-1.1.0.linux-amd64.tar.gz" \
```

Comment:
split this in 2 functions. From line 27 to line 32 we are installing `containerd`. `crio` is installed from line 18 to line 25

Reply:
Done.